v03_pipeline/lib/reference_data/clinvar.py

```diff
@@ -110,12 +112,25 @@ def parsed_and_mapped_clnsigconf(ht: hl.Table):
     )
 
 
+def get_clinvar_ht(
+    clinvar_url: str,
+    reference_genome: ReferenceGenome,
+):
+    etag = requests.head(clinvar_url, timeout=10).headers.get('ETag').strip('"')
+    try:
+        logger.info(f'Try using cached clinvar ht with etag {etag}')
+        ht = hl.read_table(clinvar_dataset_path(reference_genome, etag))
+    except hl.utils.FatalError:
+        logger.info('Cached clinvar ht not found, downloading latest clinvar vcf')
+        ht = download_and_import_latest_clinvar_vcf(clinvar_url, reference_genome)
+        ht.write(clinvar_dataset_path(reference_genome, etag), overwrite=True)
```

Comment:
I think you can use our helper `write` from `io.py` here and return the result from that method (not a huge deal though!)

Reply:
I chose not to use `write` for the submission write because I wanted to avoid the `repartition` but now since we don't need the entire clinvar ht to remain 2000 partitions I can use `write`!

Reply:
cool.  I guess, are we concerned that it'll be only one partition when we go to join the reference data though?  not the worst thing to add an arg to `write` that overrides the default partitioning.

Reply:
ah yes I added an arg to `write`. There's the concern that the join will break if the number of partitions on the clinvar table changes, because we know 2000 works and hail is hail.